webui/react/src/components/GalleryModal.tsx

```diff
@@ -67,22 +67,25 @@ const GalleryModal: React.FC<Props> = ({
     <Modal centered footer={null} open width={width} {...props}>
       <div className={css.base} style={{ minHeight }}>
         {children}
-        <IconButton
-          className={css.prev}
-          icon="arrow-left"
-          iconSize="small"
-          label="Previous"
-          tooltipPlacement="right"
-          onClick={handlePrevious}
-        />
-        <IconButton
-          className={css.next}
-          icon="arrow-right"
-          iconSize="small"
-          label="Next"
-          tooltipPlacement="left"
-          onClick={handleNext}
-        />
+        <div className={css.prev}>
```

Comment:
I think these should be the icon button from the UI Kit, no?

Reply:
So here we are using a `IconButton` which was a `Antd` Button wrapped in a `Tooltip` I instead updated the `Button` [reference to our new button](https://github.com/determined-ai/determined/pull/5726/files#diff-7f22e9343bfe613d225fedbae1c60467fb6b0152bd1d8cbd6142288f46e9ceddR5) does this sound like the correct approach?

Reply:
I misinterpreted this the first time I read it. I see what you are saying now, we have two separate icon buttons. It looks like the difference here is the existence of the tooltip. 

At first look I believe I can rewrite this to be:

```
<Tooltip ...>
   <IconButton ...>
<Tooltip ...>
```

Using UIKit components and remove the non UI Kit `IconButton` component.